modules/rostests/apitests/ntdll/RtlGetNtProductType.c

```diff
@@ -97,7 +97,7 @@ ChangeNtProductType(DWORD NtProductType)
     Result = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
                            L"SYSTEM\\CurrentControlSet\\Control\\ProductOptions",
                            0,
-                           KEY_QUERY_VALUE,
+                           KEY_ALL_ACCESS,
```

Comment:
Do you need all access because you're going to attempt writing (and not just querying) the product type value?

Reply:
Well, obviously.